File: project_euler/Problem063/Problem063.java
package project_euler;

public class Problem063 {
    public void solution() {
        int result = 0;
        for (int i = 1; i < 10; ++i) {
            result += (int) ((double) 1 / (1 - Math.log10(i)));
        }
        System.out.println(result);
    }
}

File: project_euler/Problem063/Problem063.go
package project_euler

import (
    "fmt"
    "math"
)

func Problem063() {
    result := 0
    for i := 1; i < 10; i++ {
        result += int(float64(1) / (1 - math.Log10(float64(i))))
    }
    fmt.Println(result)
}
